study-builder/studies/angio/about-you.conf

```diff
@@ -1,4 +1,5 @@
 {
+  include required("../snippets/activity-general-form.conf"),
```

Comment:
Adding an `include` so these files pull in the defaults and don't break as easily.

Reply:
This only helps if study is redeployed (re-initialized) which might happen only in developers local at this stage. 
Doesn't hurt to have them though.